File: test/MapperAI.Test/Class/Carro.java
public class Carro {
    private String marca;
    private String modelo;
    private int ano;

    public Carro(String marca, String modelo, int ano) {
        this.marca = marca;
        this.modelo = modelo;
        this.ano = ano;
    }

    public String getMarca() {
        return marca;
    }
    
    public String getModelo() {
        return modelo;
    }

    public int getAno() {
        return ano;
    }

    public void exibirInformacoes() {
        System.out.println("Marca: " + marca);
        System.out.println("Modelo: " + modelo);
        System.out.println("Ano: " + ano);
    }

    public boolean isAntigo() {
        int anoAtual = java.util.Calendar.getInstance().get(java.util.Calendar.YEAR);
        return (anoAtual - ano) > 20;
    }

}

File: test/MapperAI.Test/MappedClasses/Carro.js
namespace MapperAI.Test.MappedClasses {
  export class Carro {
    private marca: string;
    private modelo: string;
    private ano: number;

    constructor(marca: string, modelo: string, ano: number) {
      this.marca = marca;
      this.modelo = modelo;
      this.ano = ano;
    }

    public getMarca(): string {
      return this.marca;
    }

    public getModelo(): string {
      return this.modelo;
    }

    public getAno(): number {
      return this.ano;
    }

    public exibirInformacoes(): void {
      console.log(`Marca: ${this.marca}`);
      console.log(`Modelo: ${this.modelo}`);
      console.log(`Ano: ${this.ano}`);
    }

    public isAntigo(): boolean {
      const anoAtual = new Date().getFullYear();
      return (anoAtual - this.ano) > 20;
    }
  }
}
